ipatests/test_integration/test_ntp_options.py

```diff
@@ -50,7 +39,8 @@ def test_server_client_install_without_options(self):
         assert expected_msg1 in server_install.stderr_text
         assert expected_msg2 in server_install.stdout_text
 
-        client_install = self.install_client()
+        client_install = tasks.install_client(self.master, self.client)
```

Comment:
``tasks.install_client()`` returns None. You have to modify the helper function to return the result object, too.

Reply:
Will do.